controllers/controller/devworkspacerouting/solvers/resolve_endpoints.go

```diff
@@ -79,24 +80,38 @@ func getURLForEndpoint(endpoint controllerv1alpha1.Endpoint, host, basePath stri
 		protocol = controllerv1alpha1.EndpointProtocol(getSecureProtocol(string(protocol)))
 	}
 
-	// Format host/path ensuring only a single '/' character between the two. Can't use path.Join here as it would drop
-	// a trailing '/' if present
-	basehost := fmt.Sprintf("%s/%s", strings.TrimRight(host, "/"), strings.TrimLeft(basePath, "/"))
-	baseUrl := fmt.Sprintf("%s://%s", protocol, basehost)
-
-	url, err := url.Parse(baseUrl)
+	// hostWithProtocol := fmt.Sprintf("%s://%s", protocol, host)
```

Comment:
Not sure if this commented out code was intentional or not?

Reply:
Nope, left-over from refactoring